Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -25,7 +25,7 @@
 namespace DiscIO
 {
 static u32 ComputeNameSize(const File::FSTEntry& parent_entry);
-static std::string ASCIIToLowercase(std::string str);
+static std::string ASCIIToUppercase(std::string str);
```

Comment:
Are we sure that this is the best way to fix it? I mean, relying on a characters upper/lowercase translation seems not to be the appropriate way to fix it. Are there any hardware tests with filenames containing numbers? The answer could probably be as simple as ```strcasecmp``` or ```strverscmp``` with the appropriate locale?